test/Interop/SwiftToCxx/functions/cdecl.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) %s -typecheck -module-name CdeclFunctions -emit-cxx-header-path %t/empty.h
+// RUN: %FileCheck %s < %t/empty.h
+
+// RUN: %check-cxx-header-in-clang -std=c++14 %t/empty.h
+// RUN: %check-cxx-header-in-clang -std=c++17 %t/empty.h
```

Comment:
We can't automate this to run 5-10 configs?

Reply:
Good idea. I can do that as a follow-up.

Reply:
Added support here: https://github.com/apple/swift/pull/41794